apps/hash-frontend/src/pages/[shortname]/entities/[entity-uuid].page/query-editor-page.tsx

```diff
@@ -74,7 +75,7 @@ export const QueryEditorPage = (props: QueryEditorPageProps) => {
         throw new Error(res.errors?.[0]?.message ?? "Unknown error");
       }
 
-      return getRoots(res.data.results);
+      return getRoots(res.data.results) as unknown as EntityBp[];
```

Comment:
Is this still necessary now that the frontend isn't changing what `@blockprotocol/graph` it's using?